lib/ProductOpener/Display.pm

```diff
@@ -9405,59 +9406,51 @@ sub display_rev_info {
 	my $rev = shift;
 	my $code = $product_ref->{code};
 
-	my $html = '';
-
 	my $path = product_path($product_ref);
 	my $changes_ref = retrieve("$data_root/products/$path/changes.sto");
 	if (not defined $changes_ref) {
 		return '';
 	}
-	my $change_ref = $changes_ref->[$rev-1];
+
+	my $change_ref = $changes_ref->[$rev - 1];
 
 	my $date = display_date_tag($change_ref->{t});
 	my $userid = get_change_userid_or_uuid($change_ref);
-	my $user = display_tag_link("editors", $userid);
-	my $previous_link = '';
+	my $user = display_tag_link('editors', $userid);
+	my $previous_link = qw{};
+	my $product_url = product_url($product_ref);
 	if ($rev > 1) {
-		$previous_link = '/product/' . $code . '?rev='. ($rev - 1);
+		$previous_link = $product_url . '?rev='. ($rev - 1);
 	}
-	my $next_link = '';
+
+	my $next_link = qw{};
 	if ($rev < scalar @{$changes_ref}) {
-		$next_link = '/product/' . $code . '?rev=' . ($rev + 1);
+		$next_link = $product_url . '?rev=' . ($rev + 1);
 	}
 
 	my $comment = $change_ref->{comment};
 	$comment = lang($comment) if $comment eq 'product_created';
 
 	$comment =~ s/^Modification :\s+//;
 	if ($comment eq 'Modification :') {
-		$comment = '';
+		$comment = qw{};
```

Comment:
Did you mean
```suggestion
		$comment = q{};
```
Perl sees `qw{}` as a empty list:
```sh
perl -MO=Deparse -e '$comment = qw{}'

$comment = ();
-e syntax OK
```

Reply:
Why replace '' by qw{} ?

Reply:
> Why replace '' by qw{} ?

The policy [`ValuesAndExpressions::ProhibitEmptyQuotes`](https://metacpan.org/pod/Perl::Critic::Policy::ValuesAndExpressions::ProhibitEmptyQuotes) doesn't like `''`. We don't use this level right now, but I'm trying to apply it to my current changes to avoid future policy violations. 🙂